### ISSUE TITLE
abot bbox_pred

### ISSUE THREAD

**Issue by kongbia** [ohhhyeahhh/SiamCAR#8]
[    def forward(self, x):
        cls_tower = self.cls_tower(x)
        logits = self.cls_logits(cls_tower)
        centerness = self.centerness(cls_tower)
        bbox_reg = torch.exp(self.bbox_pred(self.bbox_tower(x)))](url)

in car_head.py, why add a `torch.exp` in bbox regression

**Comment by kongbia** [comment]
Besides, why is `nn.ConvTranspose2d` used when fusing multilayer features in model_builder.py

**Comment by twotwo2** [comment]
In bbox regression, we use ```torch.exp``` to ensure the outputs are positive numbers.
In this code, using ```nn.ConvTranspose2d``` or ```nn.Conv2d``` has the same effect when fusing multilayer features.

**Comment by kongbia** [comment]
> In bbox regression, we use `torch.exp` to ensure the outputs are positive numbers.
> In this code, using `nn.ConvTranspose2d` or `nn.Conv2d` has the same effect when fusing multilayer features.

Thank you for your reply!

**Comment by ghostbreaken** [comment]
> In bbox regression, we use `torch.exp` to ensure the outputs are positive numbers. In this code, using `nn.ConvTranspose2d` or `nn.Conv2d` has the same effect when fusing multilayer features.

hello, to ensure the outputs are positive numbers. why not use torch.abs to replace torch.exp?
